obspy/clients/fdsn/tests/test_client.py

```diff
@@ -861,13 +862,13 @@ def test_help_function_with_iris(self):
             # allow for changes in version number..
             got = normalize_version_number(got)
             expected = normalize_version_number(expected)
-            # catalogs/contributors are checked in separate tests
-            self.assertTrue(got[-2].startswith('Available catalogs:'))
-            self.assertTrue(got[-1].startswith('Available contributors:'))
-            got = got[:-2]
+            # convert newlines to ' '
+            got = ' '.join(got)
+            assert 'Available catalogs:' in got
+            assert 'Available contributors:' in got
             expected = expected[:-2]
-            for line_got, line_expected in zip(got, expected):
-                self.assertEqual(line_got, line_expected)
+            expected = ' '.join(expected)
+            assert got.startswith(expected)
```

Comment:
@trichter these are OK for now for the release, but I'll revert this after release is done, since it's not a bug on our end and needs to be fixed upstream =)

Reply:
I see. We can also revert right away. It is not critical for the release.